docs-conceptual/azps-6.4.0/what-is-azure-powershell.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What is Azure PowerShell
+description: This article is an introduction to Azure PowerShell and its features.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 09/13/2021
+ms.custom: devx-track-azurepowershell
+ms.service: azure-powershell
+---
+
+# What is Azure PowerShell?
+
+Azure PowerShell is a set of cmdlets for managing Azure resources directly from PowerShell. Azure
+PowerShell is designed to make it easy to learn and get started with, but provides powerful features
+for automation.
+
+## The Az PowerShell module
+
+> [!IMPORTANT]
+> The Az PowerShell module is the recommended PowerShell module for managing Azure resources on all
+> platforms.
+
+Introduced in 2018, the Az PowerShell module is based on the .NET Standard, and works with
+PowerShell 7.0.6 LTS and PowerShell 7.1.3 or higher on all platforms including Windows, macOS, and
+Linux. It's also compatible with Windows PowerShell 5.1.
+
+> [!NOTE]
+> PowerShell 7.0.6 LTS and PowerShell 7.1.3 or higher is the recommended version of PowerShell for
+> use with the Az PowerShell module on all platforms.
+
+You can install the Az PowerShell module locally on Windows, macOS, and Linux. It can also be used
+from a browser through [Azure Cloud Shell](/azure/cloud-shell/overview) or
+[inside a Docker container](/powershell/azure/azureps-in-docker). For more information, see the
+[Azure PowerShell documentation](/powershell/azure/).
+
+## The AzureRM PowerShell module
```

Comment:
I am not convinced that we need a section on AzureRM but we should have a section that talk about other modules like AzureAD or MS Graph.